cpp/velox/substrait/SubstraitToVeloxPlanValidator.cc

```diff
@@ -311,6 +311,23 @@ bool SubstraitToVeloxPlanValidator::validateIfThen(
   return true;
 }
 
+bool SubstraitToVeloxPlanValidator::validateSingularOrList(
+    const ::substrait::Expression::SingularOrList& singularOrList,
+    const RowTypePtr& inputType) {
+  auto options = singularOrList.options();
+  for (const auto& option : options) {
```

Comment:
Maybe replace `options` with `singularOrList.options()` as it is used only once.

Reply:
Updated, thanks